### DISCUSSION TITLE
Publishing to github but using a public folder on the main branch

### DISCUSSION THREAD

**Discussion by raramuridesign** [GetPublii/Publii#1880] [General] [answered]
Hi @bobmitro 
When we use a plain ftp server, we can choose the path for the project, ie public/

However when publishing to github / cloudflare, we do not have the option to publish to a folder on the main branch.

Ideally we would like to select 
+ main branch
+ output folder public/

We can set the output folder on cloudflare, but Publii, does not allow us to choose a specific folder to publish to.

Would it be possible to add this option on the git integration ?

![image](https://github.com/user-attachments/assets/3cd57f42-0c8e-4b95-a042-9cd7f18873f5)

Matt

**Comment by raramuridesign** [comment]
@dziudek Any chance you can take a look at this ?

**Reply by dziudek** [reply, marked as answer]
@raramuridesign - as I remember, we are using additional folder in Gitlab pages integration. I will try to implement such option for Github too, when I will rewrite Github integration in v.0.47

**Comment by raramuridesign** [comment]
@dziudek Thank you. This is great news. 
M.
